README.md

```diff
@@ -94,7 +94,33 @@ The building from Jupyter notebooks will NOT re-render all of the notebooks unle
 3. Add links to your content to the navigation configuration in `_quarto.yml`. For example, to link to the rendered page for `notebooks/1_cont.md`, add a link to `notebooks/1_cont.html` in `_quarto.yml`
 4. Type `quarto render` in the terminal - or use VSCode's "Render Quarto project' command using the command pallette instead.
 
+## Publish with GitHub pages
 
-***
+To publish Quarto websites to GitHub pages, you can:
+* Render sites on your local machine (see above)
+* Use the `quarto publish` command
+* Use [GitHub Actions](https://github.com/quarto-dev/quarto-actions)
 
+### Quarto publish command
+
+Enable GitHub pages in your repository settings:
+* Go to GitHub repository settings 
+* Scroll down to "GitHub Pages" section and select the following: 
+    * `Source: deploy from a branch`
+    * `Branch:gh-pages`
+    * `Save`
+
+Run the quarto publish command (assuming you are working in terminal) 
+```
+quarto publish
+```
+
+Your Quarto project will be rendered at a URL with this format:
+```html
+https://sydney-informatics-hub.github.io/<NAME-OF-YOUR-REPO>/
```

Comment:
https://pages.github.sydney.edu.au/informatics/](https://pages.github.sydney.edu.au/informatics/<NAME-OF-YOUR-REPO>/

Reply:
Im having an issue with getting the pages.github URL to work but .github.io is working. See: 
* https://sydney-informatics-hub.github.io/customising-nfcore-workshop/
* https://pages.github.sydney.edu.au/informatics/customising-nfcore-workshop/

Reply:
What are your Pages settings for the repo?

Reply:
![image](https://user-images.githubusercontent.com/73086054/222037000-f80bf350-6e4a-4c3b-bd64-189b5c90f940.png)

Reply:
My apologies for the confusion! Was following [intro to git](https://github.com/Sydney-Informatics-Hub/intro-git) setup for sake of consistency. Is there a benefit for using main vs gh-pages?

Reply:
It depends on your use and preferences. It also depends on what branch you are using as your primary development branch, but primarily I think deploying from the `main` branch gives you more flexibility in terms of site configuration and customization, since you can use any static site generator or build tool that supports GitHub pages. The `gh-pages` branch is limited to the default static site generator used by GitHub pages.

Reply:
I take that back. Fixed it!